tests/infra/remote.py

```diff
@@ -607,9 +606,6 @@ def __init__(
                 f"--notify-server-address={notify_server_host}:{notify_server_port[0]}"
             ]
 
-        if domain:
```

Comment:
Looks like a merge issue undid some changes here?

Reply:
This hasn't been rebased on a recent master, yes.